internal/controllers/import_controller.go

```diff
@@ -103,12 +86,11 @@ func (r *CAPIImportReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Ma
 		return fmt.Errorf("creating new controller: %w", err)
 	}
 
-	// Watch Rancher provisioningv2 clusters
+	// Watch Rancher provisioningv1 clusters
```

Comment:
He he, i meant v2prov when i wrote the comment and not meaning a reference to the group api version :)

This makes more sense now :)